tools/ARIAtools/vrtmanager.py

```diff
@@ -79,17 +79,16 @@ def resampleRaster(fname, cellResolution, bounds, prods_TOTbbox, outputFormat='E
 
     # Access original shape
     ds=gdal.Warp('', fname, options=gdal.WarpOptions(format="MEM", outputBounds=bounds, multithread=True, options=['NUM_THREADS=%s'%(num_threads)]))
-    arrshape=[ds.RasterYSize, ds.RasterXSize] # Get shape of full res layers
-    arrshape=[int(i*(cellResolution/100.)) for i in arrshape]
+    arrshape=[abs(ds.GetGeoTransform()[1])/(cellResolution/100.), abs(ds.GetGeoTransform()[-1])/(cellResolution/100.)] # Get output res
```

Comment:
To confirm, this does not imply integer multiple of the resolution.

Reply:
This isn't necessarily an integer multiple of the natural resolution, but the aspect ratio is preserved, and consistent vis-a-vis the other standard product layers and DEM/mask. 

I'm not sure how to provide more input flexibility than allowing to specify a percentage. E.g. if we force the input to be an integer multiple, I'm not sure of what we can allow for outside of specifying 1-arc,2-arc,4-arc, etc.